docs/source/reservations/reservations.rst

```diff
@@ -0,0 +1,208 @@
+
+.. _reservation:
+
+Reserved, Capacity Blocks & DWS
+===================================
+
+
+Clouds are designed to be elastic, but due to the recent GPU shortages, many organizations have started to get reservations
+from cloud providers to lock in GPU capacity.
+
+SkyPilot can be used to request resources from reservations and even combine them with on-demand/spot resources to fully
+utilize the capacity in your cloud accounts.
+
+.. image:: https://i.imgur.com/SbfnFDs.png
```

Comment:
How about replacing that headline with "Get GPUs from reservations & other capacity types"?

Reply:
Sounds good. Updated.